### ISSUE TITLE
wish: remember midi player position across renderings

### ISSUE THREAD

**Issue by shimpe** [wbsoft/frescobaldi#71]
I would very much enjoy a feature where the midi player can be asked to automatically return to its previous start position after re-rendering the score.

**Comment by fedelibre** [comment]
This would be very helpful. I'm arranging a tune and I'd like to check if I'm writing it correctly by looping the midi on a specific measure (or small set of measures).

The bar number indication in the MIDI pane is helpful, but dragging it over and over is boring. Being able to set up a loop would be great.

**Comment by shimpe** [comment]
Alternatively, perhaps have an option to let the midi player jump to the "point and click" position...

**Comment by wbsoft** [comment]
...if only we knew the source of the notes in the midi file. We could use heuristics, but...

maybe LilyPond could be programmed to write textedit messages in the MIDI :-)

**Comment by uliska** [comment]
Surely it could. But then no tool except Frescobaldi could play back the MIDI files ;-)

**Comment by wbsoft** [comment]
No, you can embed messages in the MIDI that are safely ignored by applications that do not understand them.

**Comment by fedelibre** [comment]
> ...if only we knew the source of the notes in the midi file. We could use heuristics, but...
> 
> maybe LilyPond could be programmed to write textedit messages in the MIDI :-)

here's the LilyPond issue:
https://sourceforge.net/p/testlilyissues/issues/2473/
